tmt.spec

```diff
@@ -90,6 +90,20 @@ Recommends:     qemu-system-x86-core
 
 %description -n tmt+provision-virtual %_metapackage_description
 
+%package -n     tmt+provision-bootc
```

Comment:
RPM build fails with the following:

```
Processing files: tmt+provision-bootc-1.39.dev888-1.20241106153118869510.pr3161.23.g7d63c518.fc39.noarch

Error: The package name contains an extras name `provision-bootc` that was not found in the metadata.
Check if the extras were removed from the project. If so, consider removing the subpackage and obsoleting it from another.

error: Dependency tokens must begin with alpha-numeric, '_' or '/': *** PYTHON_EXTRAS_NOT_FOUND_ERROR___SEE_STDERR ***

Error: The package name contains an extras name `provision-bootc` that was not found in the metadata.
Check if the extras were removed from the project. If so, consider removing the subpackage and obsoleting it from another.

error: Dependency tokens must begin with alpha-numeric, '_' or '/': *** PYTHON_EXTRAS_NOT_FOUND_ERROR___SEE_STDERR ***
```

I'm not an expert on rpmbuild, but IIUIC, the problem is no `provision-bootc` extras in `pyproject.toml`. Even an empty one, like `provision-container`, could resolve the problem: https://github.com/teemtee/tmt/blob/main/pyproject.toml#L62

Reply:
thanks, pushed the change to pyproject.tml